scripts/azure-pipelines/build_tomviz.sh

```diff
@@ -2,7 +2,9 @@
 
 if [[ $AGENT_OS == 'Darwin' ]]; then
   # CMake can't find Qt on mac os unless we do this
-  CMAKE_EXTRA_ARGS="-DCMAKE_PREFIX_PATH=/usr/local/opt/qt/"
+  CMAKE_EXTRA_ARGS="-DCMAKE_PREFIX_PATH=/usr/local/opt/qt/ "
+  # Resort to brute force to get CMake to find the right library!
+  CMAKE_EXTRA_ARGS+="-DPython3_LIBRARY=/Users/runner/hostedtoolcache/Python/3.8.7/x64/lib/libpython3.8.dylib "
```

Comment:
Let's hope they don't change this soon. They might if a new patch such as python 3.8.8 comes out.

Reply:
If the python3 binary is in the same dir, something like this might work too, that avoids the patch number:
```
  CMAKE_EXTRA_ARGS+="-DPython3_LIBRARY=$(which python)/../lib/libpython3.8.dylib "

```

Reply:
@psavery If you would like to try that feel free to push a commit.